CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.2.0] - 2018-06-04
```

Comment:
worth to add actual link to (non-existing yet) 1.2.0 release on github (https://github.com/grpc-ecosystem/go-grpc-prometheus/releases/tag/v1.2)

Reply:
I agree this comment.